tables/Tables/BaseTabIter.h

```diff
@@ -147,7 +149,8 @@ class BaseTableIterator
     std::shared_ptr<Vector<size_t>> sortIterKeyIdxChange_p;
     Vector<rownr_t>::iterator sortIterBoundariesIt_p;
     Vector<size_t>::iterator  sortIterKeyIdxChangeIt_p;
-    RefTable* aRefTable_p;
+    std::shared_ptr<BaseTable> aBaseTable_p;
+    RefTable* aRefTable_p;      //# Same as aBaseTable_p
```

Comment:
See my later/earlier comment about enable_shared_from_this, would that be possible here maybe too?